src/arm/bbai-bone-buses.dtsi

```diff
@@ -468,8 +468,15 @@ bone_timer_5: &timer15 {
 
 // McASP
 bone_mcasp_0: &mcasp1 {
+        #sound-dai-cells= <0>;
```

Comment:
Indentation looks inconsistent. Let's try using tabs. If you need to do a formatting fix, do that separate from functional fixes.

Reply:
Indentation problem again 😂 I always try my best to resolve them but one always slips in.